quic/s2n-quic-core/src/recovery/bbr.rs

```diff
@@ -842,7 +889,10 @@ impl BbrCongestionController {
         // This applies regardless of whether packet conservation is in place, as the pseudocode
         // applies this clamping within BBRBoundCwndForModel(), which is called after all prior
         // cwnd adjustments have been made.
-        self.cwnd = cwnd.clamp(self.minimum_window(), self.bound_cwnd_for_model());
+        self.cwnd = cwnd.clamp(
+            BbrCongestionController::minimum_window(self.max_datagram_size),
```

Comment:
nit: I think you can add a `use BbrCongestionController::minimum_window` so you can just call `minimum_window`, little more readable

Reply:
or at least just call `Self::minimum_window`